src/Tizen.NUI/src/public/Common/TimePeriod.cs

```diff
@@ -108,5 +103,12 @@ private int SecondsToMilliSeconds(float sec)
         {
             return (int)(sec * 1000);
         }
+
```

Comment:
API-Bot gave "ACR Required" tag but this `ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef)` should be hidden-api, so it is OK.